include/CL/sycl.hpp

```diff
@@ -1,28 +1,12 @@
 /*
- * This file is part of hipSYCL, a SYCL implementation based on CUDA/HIP
+ * This file is part of AdaptiveCpp (formerly hipSYCL, Open SYCL), an
```

Comment:
Do you explicitly want to keep the old project names here? The project name is "AdaptiveCpp" now, and the file is part of it. It is not part of hipSYCL anymore (since hipSYCL is no more).

In general, I myself prefer to mention the original name when referring to the project, for clarity. But I am not sure this is helpful in copyright headers.

Reply:
I agree with your point. If @illuhad has nothing against it, I remove the hipSYCL/ Open SYCL mentions.